README.md

```diff
@@ -117,7 +123,69 @@ MIDDLEWARE = [
 ]
 ```
 
-if you do like to use admin interface  in your app, when using this module, you will also need to install and configure the [custom_usermodel](https://github.com/uktrade/django-staff-sso-usermodel).
+## Change the default user id field
+
+Staff-sso maintains two unique user ids for each user: the `email_user_id` field, which is in an email format [NOTE: it is purely a unique id, not a valid email address] and the `user_id` field, which is a GUID.  By default (from version 3.0.0 onwards) django-staff-sso-client identifies users based on the `email_user_id` field.  This is the preferred option for most cases.  If however, you need to use the `user_id` field, then add this to your settings.py file:
```

Comment:
Did you decide against sso_user_id and sso_user_id_guid in the end? Perhaps you want to keep it consistent with how they are named in SSO.

Reply:
Yes, I did want to keep it consistent.  Also the configuration means that this field isn't really directly exposed... the email_user_id field is used behind the scenes automatically, with the option to use the guid with the setting:

`AUTHBROKER_USE_USER_GUID=True`

I'm open to changing this though if you think it would make it easier to understand? But at this point I think we'd just be looking at changing the field names in the documentation.  Let me know what you think.

Reply:
I think this is a good approach and does not need to change.